vscode_extension/src/extension.ts

```diff
@@ -1,14 +1,16 @@
-import * as vscode from 'vscode';
-import { AacTaskProvider } from './AacTaskProvider';
+import { Disposable, ExtensionContext, tasks } from "vscode";
+import { AacLanguageServerClient } from "./AacLanguageServer";
+import { AacTaskProvider } from "./AacTaskProvider";
 
-let aacTaskProvider: vscode.Disposable | undefined;
+let aacTaskProvider: Disposable | undefined;
+let aacLspClient: AacLanguageServerClient = AacLanguageServerClient.makeLspClient();
```

Comment:
Related to the other LspClient comment.
```suggestion
let aacLspClient: AacLanguageServerClient = AacLanguageServerClient.getLspClient();
```

Reply:
Resolved in 772c0c221d1181eeb3b81f48c878cb2e6a07127f